packages/flutter/lib/src/painting/_load_asset_manifest_io.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+/// Loads the contents of the asset manifest generated at build time.
+// This implementation, for non-web apps, is null since loading the asset
+// manifest has a higher probability of working.
+// We want to use the same signature as the web counterpart, so we
+// declare this as final rather than const.
+// ignore: prefer_const_declarations
```

Comment:
This comment seems out of date

Reply:
Addressed in commit [fix documented [sic.] of loadAssetManifest in _load_asset_manifest_io.dart](https://github.com/flutter/flutter/pull/130419/commits/2bf389bd6403fca34e62783a0ad50be490b6a2f3)